profile/README.md

```diff
@@ -0,0 +1,24 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+  <img align="center" src="/profile/logo.png" height="30%" width="30%" />
```

Comment:
Can we please take this image directly from the org profile image?

Reply:
No

Reply:
@Ayush7614 any specific reason why we can't directly use https://avatars.githubusercontent.com/u/92252339?s=200&v=4

Reply:
> @Ayush7614 any specific reason why we can't directly use https://avatars.githubusercontent.com/u/92252339?s=200&v=4

oh you can say this image i can add this now. Now you can review